<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>AdvancedDialogImpl</class>
  <widget class="QWidget" name="AdvancedDialogImpl" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>403</width>
        <height>399</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <item row="1" column="0" >
        <widget class="Q3GroupBox" name="qgbPriority" >
          <property name="title" >
            <string>Screen Saver Priority</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item row="0" column="0" >
              <widget class="QComboBox" name="qcbPriority" >
                <property name="toolTip" stdset="0" >
                  <string/>
                </property>
                <item>
                  <property name="text" >
                    <string>Low</string>
                  </property>
                </item>
                <item>
                  <property name="text" >
                    <string>Medium</string>
                  </property>
                </item>
                <item>
                  <property name="text" >
                    <string>High</string>
                  </property>
                </item>
              </widget>
            </item>
            <item row="0" column="1" >
              <spacer name="spacer5" >
                <property name="sizeHint" >
                  <size>
                    <width>31</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Horizontal</enum>
                </property>
              </spacer>
            </item>
          </layout>
        </widget>
      </item>
      <item row="0" column="0" >
        <widget class="Q3GroupBox" name="groupBox1" >
          <property name="title" >
            <string>Screen Corner Actions</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item rowspan="2" row="3" column="0" colspan="2" >
              <widget class="QLabel" name="qlMonitorLabel" >
                <property name="text" >
                  <string/>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="0" column="0" colspan="2" >
              <layout class="QHBoxLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item>
                  <layout class="QVBoxLayout" >
                    <property name="margin" >
                      <number>0</number>
                    </property>
                    <item>
                      <widget class="QLabel" name="qlTopLeft" >
                        <property name="text" >
                          <string>Top left:</string>
                        </property>
                        <property name="wordWrap" >
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QComboBox" name="qcbTopLeft" >
                        <item>
                          <property name="text" >
                            <string>No Action</string>
                          </property>
                        </item>
                        <item>
                          <property name="text" >
                            <string>Lock Screen</string>
                          </property>
                        </item>
                        <item>
                          <property name="text" >
                            <string>Prevent Locking</string>
                          </property>
                        </item>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item>
                  <spacer name="spacer6" >
                    <property name="sizeHint" >
                      <size>
                        <width>41</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item>
                  <layout class="QVBoxLayout" >
                    <property name="margin" >
                      <number>0</number>
                    </property>
                    <item>
                      <widget class="QLabel" name="qlTopRight" >
                        <property name="text" >
                          <string>Top right:</string>
                        </property>
                        <property name="wordWrap" >
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QComboBox" name="qcbTopRight" >
                        <item>
                          <property name="text" >
                            <string>No Action</string>
                          </property>
                        </item>
                        <item>
                          <property name="text" >
                            <string>Lock Screen</string>
                          </property>
                        </item>
                        <item>
                          <property name="text" >
                            <string>Prevent Locking</string>
                          </property>
                        </item>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
            </item>
            <item rowspan="1" row="5" column="0" colspan="2" >
              <layout class="QHBoxLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item>
                  <layout class="QVBoxLayout" >
                    <property name="margin" >
                      <number>0</number>
                    </property>
                    <item>
                      <widget class="QLabel" name="qlBottomLeft" >
                        <property name="text" >
                          <string>Bottom left:</string>
                        </property>
                        <property name="wordWrap" >
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QComboBox" name="qcbBottomLeft" >
                        <item>
                          <property name="text" >
                            <string>No Action</string>
                          </property>
                        </item>
                        <item>
                          <property name="text" >
                            <string>Lock Screen</string>
                          </property>
                        </item>
                        <item>
                          <property name="text" >
                            <string>Prevent Locking</string>
                          </property>
                        </item>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item>
                  <spacer name="spacer7" >
                    <property name="sizeHint" >
                      <size>
                        <width>20</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item>
                  <layout class="QVBoxLayout" >
                    <property name="margin" >
                      <number>0</number>
                    </property>
                    <item>
                      <widget class="QLabel" name="qlBottomRight" >
                        <property name="text" >
                          <string>Bottom right:</string>
                        </property>
                        <property name="wordWrap" >
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QComboBox" name="qcbBottomRight" >
                        <item>
                          <property name="text" >
                            <string>No Action</string>
                          </property>
                        </item>
                        <item>
                          <property name="text" >
                            <string>Lock Screen</string>
                          </property>
                        </item>
                        <item>
                          <property name="text" >
                            <string>Prevent Locking</string>
                          </property>
                        </item>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
            </item>
            <item rowspan="1" row="2" column="0" colspan="2" >
              <layout class="QHBoxLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item>
                  <spacer name="spacer8" >
                    <property name="sizeHint" >
                      <size>
                        <width>21</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item>
                  <widget class="QLabel" name="monitorLabel" >
                    <property name="text" >
                      <string/>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="spacer9" >
                    <property name="sizeHint" >
                      <size>
                        <width>31</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Horizontal</enum>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
            <item row="1" column="1" >
              <spacer name="spacer10" >
                <property name="sizeHint" >
                  <size>
                    <width>20</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Vertical</enum>
                </property>
              </spacer>
            </item>
            <item row="4" column="1" >
              <spacer name="spacer11" >
                <property name="sizeHint" >
                  <size>
                    <width>20</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Vertical</enum>
                </property>
              </spacer>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
